data-prepper-plugins/otel-proto-common/src/test/resources/test-request-multiple-logs.json

```diff
@@ -10,59 +10,34 @@
         }
       }]
     },
-    "scopeLogs": [{
-      "logRecords": [{
-        "timeUnixNano": "1590328800000000000",
-        "severityNumber": "SEVERITY_NUMBER_DEBUG",
-        "severityText": "Severity value",
-        "body": {
-          "stringValue": "Log value"
-        },
-        "attributes": [{
-          "key": "statement.params",
-          "value": {
-            "stringValue": "us-east-1"
+    "scopeLogs": [
+      {
+        "logRecords": [
+          {
+            "timeUnixNano": "1590328800000000000",
+            "severityNumber": "SEVERITY_NUMBER_DEBUG",
+            "severityText": "Severity value",
+            "body": {
+              "stringValue": "Log value"
+            },
+            "attributes": [
+              {
+                "key": "statement.params",
+                "value": {
+                  "stringValue": "us-east-1"
+                }
+              }
+            ],
+            "droppedAttributesCount": 3,
+            "flags": 1,
+            "traceId": "uhocI7QJO2M=",
+            "spanId": "LMg6yQ68Rpw=",
+            "observedTimeUnixNano": "1590328802000000000"
           }
-        }],
-        "droppedAttributesCount": 3,
-        "flags": 1,
-        "traceId": "uhocI7QJO2M=",
-        "spanId": "LMg6yQ68Rpw=",
-        "observedTimeUnixNano": "1590328802000000000"
-      }]
-    }],
+        ]
+      }
+    ],
     "schemaUrl": "schemaurl"
-    },
-    {
-    "resource": {
```

Comment:
Why "resource" section removed? I think it is still valid in the new protocol spec, right?